src/saveload/saveload.cpp

```diff
@@ -1437,7 +1433,7 @@ size_t SlCalcObjMemberLength(const void *object, const SaveLoad &sld)
 		case SL_REF:
 		case SL_ARR:
 		case SL_STR:
-		case SL_LST:
+		case SL_REFLST:
```

Comment:
Since you're renaming & not preserving length, why not just go for the full `SL_REFLIST` ?